src/components/oInput.js

```diff
@@ -197,14 +235,23 @@ oInput.prototype.readOnly = function(readonly) {
 }
 
 oInput.prototype.required = function(required) {
-  if(typeof required !== 'boolean') return this;
+  if(required !== 'true' && required !== 'false' && typeof required !== 'boolean') return this;
```

Comment:
Tu jest to required co do disabled się wkradło :D Tu to samo co z readonly i disabled